bin/mail/2016-09-15-latam-hoc-starwars/generate-mailing-lists.rb

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+
+# Query all HOC organizers and Code Studio teachers from any of the Latam countries.
+
+# Latin America (latam)
+LATAM_ES_COUNTRIES = [
+  'Mexico',
+  'Guatemala',
+  'El Salvador',
+  'Costa Rica',
+  'Nicaragua',
+  'Belize',
+  'Panama',
+  'Cuba',
+  'Dominican Republic',
+  'Venezuela',
+  'Colombia',
+  'Ecuador',
+  'Peru',
+  'Bolivia',
+  'Paraguay',
+  'Uruguay',
+  'Chile',
+  'Argentina',
+  'Puerto Rico'
```

Comment:
Any reason these can't be alphabetized? Makes confirming you aren't missing any easier...

Do you want Honduras (missing)? Do you want Belize (extra)? Do you want the other tiny countries (missing)?

https://en.wikipedia.org/wiki/Latin_America
https://en.wikipedia.org/wiki/List_of_Latin_American_countries_by_population